fdbserver/KeyValueStoreRocksDB.actor.cpp

```diff
@@ -359,6 +370,17 @@ class RocksDBErrorListener : public rocksdb::EventListener {
 	std::mutex mutex;
 	UID id;
 };
+
+class RocksDBEventListener : public rocksdb::EventListener {
+public:
+	RocksDBEventListener(std::shared_ptr<double> lastFlushTime) : lastFlushTime(lastFlushTime){};
+
+	void OnFlushCompleted(rocksdb::DB* db, const rocksdb::FlushJobInfo& info) override { *lastFlushTime = now(); }
+
+private:
+	std::shared_ptr<double> lastFlushTime;
```

Comment:
Consider using atomic? lastFlushTime is used by two threads.

Reply:
I thought of using it. One Thread is reading the value and the other thread is updating the value. If the reader thread gets a wrong value, that will be still fine in this case(probably an extra flush or no flush). Considering the cost of atomic, avoided it here in this case.

Reply:
ok. Please add a comment. Just in case we got thread sanitizer complains.

Reply:
sure. I will remember it add it in next diff, don't want to block the release with this comment.